torchgeo/losses/qr_losses.py

```diff
@@ -0,0 +1,53 @@
+"""Loss functions for learing on the prior."""
```

Comment:
What does the `qr` in the filename stand for? Maybe we could use that as the filename instead, the fact that it's in the `losses` directory implies that it's a loss. For example, we put datasets in `torchgeo/datasets/landsat.py`, not `torchgeo/datasets/landsat_datasets.py`

Reply:
These are the names of the distributions used in the loss

Reply:
Can rename the file to `qr.py` or `prior.py`? I think it's already clear that these are losses from the `torchgeo.losses` or from the class names.